src/features/search/components/SearchResults.tsx

```diff
@@ -40,12 +44,23 @@ export const SearchResults: React.FC = () => {
     isFetchingNextPage,
   } = useSearchResults()
   const { searchState } = useSearch()
+  const { dispatch } = useStagedSearch()
   const showSkeleton = useIsFalseWithDelay(isLoading, ANIMATION_DURATION)
   const isRefreshing = useIsFalseWithDelay(isFetching, ANIMATION_DURATION)
   const isFocused = useIsFocused()
 
   const { headerTransition: scrollButtonTransition, onScroll } = useOpacityTransition()
 
+  const { navigate } = useNavigation<UseNavigationType>()
+  const { locationFilter } = searchState
+  const { locationType } = locationFilter
+  // PLACE and VENUE belong to the same section
+  const section = locationType === LocationType.VENUE ? LocationType.PLACE : locationType
+  const { label: locationLabel } = useLocationChoice(section)
+  const { data: appSettings } = useAppSettings()
+  // Feature flag use to manage display filters buttons on search results view
+  const appEnableCategoryFilterPage = appSettings?.appEnableCategoryFilterPage ?? false
```

Comment:
j'ai l'impression que ce commentaire est présent pour expliciter du code qui n'est pas assez explicite

est-ce que si on renommait la variable ça serait suffisament explicite ?

proposition :
```suggestion
  const displayFiltersButtons = appSettings?.appEnableCategoryFilterPage ?? false
```